carbonplan_project/__init__.py

```diff
@@ -1,7 +1,16 @@
 import pathlib
+from importlib.metadata import PackageNotFoundError as _PackageNotFoundError
+from importlib.metadata import version as _version
 
 from intake import open_catalog
 
+try:
+    __version__ = _version(__name__)
+except _PackageNotFoundError:
+    # package is not installed
+    __version__ = "unknown"
+
+
 cat_dir = pathlib.Path(__file__)
```

Comment:
I haven't looked at the catalog import, but I'm guessing it's probably out of date. Should we remove/comment out this for now or just leave it in?